src/Tribe/Main.php

```diff
@@ -416,7 +416,7 @@ public function inject_tickets_json_ld( $data, $args, $post ) {
 	 * @return void
 	 */
 	public function add_linking_archor() {
-		_deprecated_function( __METHOD__, '4.4.8', 'Tribe__Tickets__Main::add_linking_anchor' );
+		_deprecated_function( __METHOD__, 'TBD', 'Tribe__Tickets__Main::add_linking_anchor' );
```

Comment:
We did indeed deprecate this in 4.4.8 [(see here)](https://github.com/moderntribe/event-tickets/blob/4.4.8/src/Tribe/Main.php#L413) ... seems like we shouldn't be changing it to a higher version number at this point?